packages/dashboard/src/components/Card/Layout/Layout.tsx

```diff
@@ -102,20 +106,24 @@ export class Layout extends React.Component<ILayoutProps> {
                   data,
                   chartType,
                   dataPoints,
-                  compactChartWidth
+                  compactChartWidth,
+                  chartUpdatedOn
                 } = cardContent.content as IChartProps;
                 contentArea.push(
-                  <Chart
-                    chartLabels={chartLabels}
-                    chartType={chartType}
-                    legendColors={legendColors}
-                    barWidth={barWidth}
-                    data={data}
-                    dataPoints={dataPoints}
-                    compactChartWidth={compactChartWidth}
-                    width={this._getChartWidth(cardContentList.length)}
-                    height={this._getChartHeight(cardContentList.length)}
-                  />
+                  <React.Fragment>
+                    {chartUpdatedOn && <div className={dataVizLastUpdateClassName}>{chartUpdatedOn}</div>}
```

Comment:
Just a non-blocking suggestion to consider [\<time>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time) with the `datetime` attribute and optionally `title` for hover (Reddit post timestamps do this).

Reply:
Currently that design team wanted this to be a free text to give the flexibility to show something like 'last updated today' or 'last updated yesterday'.